### ISSUE TITLE
module does not load in AC

### ISSUE THREAD

**Issue by Jaunius343** [jesung/aidriver#1]
module does not load in AC.
so far I got two errors in logs. one with ctypes, which i solved using  third party and siminfo from https://github.com/huntervaners/Template_Assetto_Corsa_App

another issue is with sockets.
  File "E:\SteamLibrary\SteamApps\common\assettocorsa\system\x64\python33.zip\socket.py", line 47, in <module>
ImportError: No module named '_socket'

ctypes error was the same, it could not find _ctypes in python33.zip\ctypes.py

I did a fresh install of AC on my second pc to test, the module still does not load, could not get the error, because py_logs.txt is empty (maybe because content manager is not installed) I will check later if errors match on both pcs.

**Comment by jesung** [comment]
Thanks for flagging. I had to reinstall my OS so I will try to replicate the issue and get back to you

**Comment by Jaunius343** [comment]
ok. so py_log.txt was always re-initialized empty, until i installed custom shaders patch on content manager.
this is the output i got (log below). No idea why it's missing references to _socket and _ctypes as they should be included in python package. I did a clean install specifically to see if AC python was messed up on my pc. Also, funny that ctypes are getting initialized only once.  Template_Assetto_Corsa_App has a different implementation of ctypes, but is still pointing to aidriver directory. 

[aidriver: error] Traceback (most recent call last):
  File "apps/python/aidriver\aidriver.py", line 6, in <module>
    import socket
  File "C:\Program Files (x86)\Steam\steamapps\common\assettocorsa\system\x64\python33.zip\socket.py", line 47, in <module>
ImportError: No module named '_socket'

[Template_Assetto_Corsa_App: error] Traceback (most recent call last):
  File "apps/python/Template_Assetto_Corsa_App\Template_Assetto_Corsa_App.py", line 14, in <module>
    from third_party.sim_info import *
  File "apps/python/aidriver\third_party\sim_info.py", line 29, in <module>
    import ctypes
  File "C:\Program Files (x86)\Steam\steamapps\common\assettocorsa\system\x64\python33.zip\ctypes\__init__.py", line 7, in <module>
ImportError: No module named '_ctypes'

**Comment by jesung** [comment]
Confirming that I'm getting the same error as you on the fresh install of AC after installing Content Manager with Custom Shader Patch. Will tinker with this in the next few days

**Comment by Jaunius343** [comment]
I managed to get it running by installing python 3.3.5 on my machine. Assetto corsa no longer throws an error.
Just had to change one import, because it couldn't find the package.
"from sim_info import info" to "from third_party.sim_info import info"

And a question regarding [ACSelfDriving](https://github.com/jesung/ACSelfDriving)
I noticed on your [blog](https://jesungpark.com/projects/sdrc1.html) that you set specific steering speed and steering gamma on AC
![image](https://github.com/jesung/aidriver/assets/73691989/c1646514-b5ea-40df-8745-74fc208a053b)

could you share these values? 

Base AC settings were:
steering speed - 20%
steering gamma 200%
(car goes off-track with these.)

With some testing done, steering speed does not seem to have much impact, but gamma does
steering speed - 20%
steering gamma 100%
(car completes a full clean lap in 1:07)

**Comment by jesung** [comment]
Confirming that installing Python 3.3.5 got things working again. Thanks a lot for helping to debug!
It's not ideal to require a specific Python installation locally so I was trying out individual _socket.pyd files but couldn't get it to work.

Just did a 1:06.8 with the below settings. Goal is to have the controller inputs translate 1:1 with the game at all times because the steering angle formula is not dependent on speed, current steering angle, sudden inputs, etc.
speed sensitivity: 0%
steering speed: 100%
steering gamma: 100%
steering filter: 0%
